bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -8,97 +8,127 @@
 		<item-type>Number:Pressure</item-type>
 		<label>Air Pressure</label>
 		<description>Air pressure in hPa</description>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+		<state readOnly="true" pattern="%.1f hPa"/>
```

Comment:
What's the reason for replacing the units by constants?

Reply:
This however did solve the above problem, so I figured I'd set it explicitly for all channels, so there's a sensible default. I can still be overridden by setting the state description on the Item to a different unit. If I use %unit% for the channels, the state description would have to be set for each Item to make it display the correct unit, which can be a lot of work for the end user. 

Example:
Without explicit unit:
```
openhab> openhab:status SMHIWeatherForecast_MinWindSpeed
5.759990784011058554102274358886417 km/h
```
With explicit unit:
```
openhab> openhab:status SMHIWeatherForecast_MinWindSpeed
1.6 m/s
```
Should this be considered a bug, and an issue raised in openhab-core?

Reply:
The units might be converted by the framework due to locale settings. As it's unlikely that this binding will be used by users with imperial units, I think it's good to go with constant units.